PhysicsTools/NanoAOD/python/isotracks_cff.py

```diff
@@ -3,11 +3,10 @@
 
 finalIsolatedTracks = cms.EDProducer("IsolatedTrackCleaner",
     tracks = cms.InputTag("isolatedTracks"),
-    cut = cms.string("pt > 10 && abs(dxy) < 0.02 && abs(dz) < 0.1 && isHighPurityTrack && miniPFIsolation.chargedHadronIso/pt < 0.2"), 
+    cut = cms.string("((pt>5 && (abs(pdgId) == 11 || abs(pdgId) == 13)) || pt > 10) && (abs(pdgId) < 15 || abs(eta) < 2.5) && abs(dxy) < 0.2 && abs(dz) < 0.1 && ((pfIsolationDR03().chargedHadronIso < 5 && pt < 25) || pfIsolationDR03().chargedHadronIso/pt < 0.2)"), 
```

Comment:
what's the sense of `abs(pdgId) < 15 || abs(eta) < 2.5` ?

Reply:
Most isolated track vetoes are made without an explicit eta cut.  Looking at some nanoAOD events in ttbar MC revealed strange spikes in isolated pf candidates that are not leptons at high eta which contributed significantly to the rate and could not correspond to the isolated prompt candidates that we are interested in.  An eta cut for non-lepton pf candidates was added to suppress these.  In principle, I think it would be fine to just make the eta cut < 2.5 always.